docs/develop/json-manifest-overview.md

```diff
@@ -74,19 +74,19 @@ The base manifest properties specify characteristics of the add-in that *any* ty
 
 |JSON property|Purpose|XML element(s)|Comments|
```

Comment:
Per the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=25525), we should use the plural form of the noun when the number is variable. I've also found that the narrator read this as "element ess".
```suggestion
|JSON property|Purpose|XML elements|Comments|
```